app/models/pageflow/account.rb

```diff
@@ -18,6 +18,7 @@ class Account < ActiveRecord::Base
     def build_default_theming(*args)
       super.tap do |theming|
         theming.account = self
+        theming.theme_name = "default"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.